src/routes/MapExplorer/MapExplorer.js

```diff
@@ -252,13 +273,66 @@ export default class MapExplorer extends React.Component {
     this.setState({ showFeatureFilters: !this.state.showFeatureFilters })
   }
 
+  toggleSavedSearches = (e) => {
+    this.setState({ showSavedSearches: !this.state.showSavedSearches })
+  }
+
+  saveSearch = (name) => {
+    const { currentBoundingBox } = this.state
+    const { currentUser, filters, location, posts } = this.props
+    const { featureTypes, search: searchText, topics } = filters
+    const { pathname } = location
+
+    let context
```

Comment:
Ah yes - this can indeed be pulled out of `routeParams.context`, and @tibetsprague made some changes to `MapExplorer.connector` that maps the context to subject. I'll remove this